src/components/Collections/CreateCollections.svelte

```diff
@@ -1186,42 +1144,33 @@
         </div>
 
         <!-- Description -->
-        <div
-          class="mb-16 flex w-[99.1%] flex-col items-start justify-start gap-[7px]"
-        >
-          <div class="m-0 w-full items-center p-0">
-            <span class="mr-[13px] font-body">Description</span>
-            <span class="font-body text-[#EB48F8]"> * </span>
-          </div>
-          <label class="hs-form-field is-filled">
-            <textarea
+        <label class="hs-form-field is-filled is-required">
+          <span class="hs-form-field__label">Description</span>
+          <textarea
             class="hs-form-field__input"
             bind:value={membership.description}
             on:change={updateState}
             id="membership-description"
             name="membership-description"
             disabled={membershipExists}
-            />
+          />
           <p class="hs-form-field__helper">Markdown is available</p>
-          </label>
-        </div>
+        </label>
 
         <!-- Save & Delete Buttons -->
-        <div class="mb-16 flex items-start gap-16">
+        <div class="mb-8 flex items-start gap-16">
           <button
-            on:click={() => handleSaveClick()}
-            type="button"
-            class={`hs-button is-large is-filled w-fit rounded px-8 py-6 text-base font-bold text-white`}
-            >
+            on:click={() => update()}
```

Comment:
Hi, thanks for the PR, this should be,
```suggestion
           on:click={() => handleSaveClick()
```
Since, handleSaveClick updates local states correctly, for appending new membership addition in local collection state
```ts
  const handleSaveClick = () => {
    updateState()
    update()
    setIsAdding(false)
    membership = {
      ...defaultMembership,
    }
    onChangePrice(membership)
  }
```